queries/omen/markets.ts

```diff
@@ -250,6 +250,7 @@ const getMarketTransactionsQuery = gql`
       }
       fpmm {
         collateralToken
+        outcomes
```

Comment:
We can delete this, right?

Reply:
Yes! Thanks for the heads up! Forgot to clean this up after consuming the trades only